jupyterlab-server-proxy/package.json

```diff
@@ -24,21 +24,35 @@
     "url": "https://github.com/jupyterhub/jupyter-server-proxy.git"
   },
   "scripts": {
-    "build": "tsc",
-    "clean": "rimraf lib",
-    "prepare": "npm run clean && npm run build",
-    "watch": "tsc -w"
+    "build": "jlpm run build:lib && jlpm run build:labextension:dev",
+    "build:prod": "jlpm run build:lib && jlpm run build:labextension",
+    "build:labextension": "jupyter labextension build .",
+    "build:labextension:dev": "jupyter labextension build --development True .",
+    "build:lib": "tsc",
+    "clean": "jlpm run clean:lib",
+    "clean:lib": "rimraf lib tsconfig.tsbuildinfo",
+    "clean:labextension": "rimraf {{ cookiecutter.python_name }}/labextension",
+    "clean:all": "jlpm run clean:lib && jlpm run clean:labextension",
+    "eslint": "eslint . --ext .ts,.tsx --fix",
+    "eslint:check": "eslint . --ext .ts,.tsx",
+    "install:extension": "jupyter labextension develop --overwrite .",
+    "prepare": "jlpm run clean && jlpm run build:prod",
+    "watch": "run-p watch:src watch:labextension",
+    "watch:src": "tsc -w",
+    "watch:labextension": "jupyter labextension watch ."
   },
   "dependencies": {
     "@jupyterlab/application": "^1.0.0 || ^2.0.0 || ^3.0.0",
     "@jupyterlab/apputils": "^1.0.0 || ^2.0.0 || ^3.0.0",
     "@jupyterlab/launcher": "^1.0.0 || ^2.0.0 || ^3.0.0"
   },
   "devDependencies": {
+    "@jupyterlab/builder": "^3.0.2",
     "rimraf": "^2.6.1",
     "typescript": "~3.7.0"
   },
   "jupyterlab": {
-    "extension": true
+    "extension": true,
+    "outputDir": "../jupyter_server_proxy/labextension"
```

Comment:
This is equivalent to how [jupytext does it](https://github.com/mwouts/jupytext/blob/master/packages/labextension/package.json#L57)